examples/promise-co-await/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "stage-2"],
```

Comment:
can you help with this? I think I just threw here first version that worked with aync/await